front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -54,6 +54,7 @@ const StcdmResults = ({
   const hasSimulationResults = hasResults(outputs);
 
   const { trackConflicts, workConflicts } = useConflictsMessages(t, outputs);
+  const { username } = useAuth();
```

Comment:
You can move this hook directly in StdcmReportSheet

Reply:
i've encountered an [issue](https://stackoverflow.com/questions/77947140/cannot-call-state-from-redux-in-react-pdf-component) doing that of redux being out of scope of the <Provider>